library/java/io/envoyproxy/envoymobile/Envoy.java

```diff
@@ -1,46 +1,14 @@
 package io.envoyproxy.envoymobile;
 
 import android.content.Context;
-import android.net.ConnectivityManager;
-import android.util.Log;
-
-import java.io.*;
+import io.envoyproxy.envoymobile.EnvoyEngine;
 
 // Wrapper class that allows for easy calling of Envoy's JNI interface in native Java.
 public class Envoy {
```

Comment:
Are we going to port to Kotlin in a separate PR?

Reply:
Nevermind, I see the implementation below.

Reply:
So the Kotlin version looks like this: https://github.com/lyft/envoy-mobile/pull/158/files#diff-6f7df3634fc91d9c4910cb9040b88705

I was going to delete the Java version after you got a chance to compare